provisioner/influx-series.js

```diff
@@ -39,6 +39,78 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
   },
 });
 
+// Store the times when a spot request is fulfilled
+module.exports.spotRequestsFulfilled = new base.stats.Series({
+  name: 'AwsProvisioner.SpotRequestsFulfilled',
+  columns: {
+    provisionerId: base.stats.types.String,
+    region: base.stats.types.String,
+    az: base.stats.types.String,
+    instanceType: base.stats.types.String,
+    workerType: base.stats.types.String,
+    id: base.stats.types.String,
+    instanceId: base.stats.types.String,
+    // This is the true time per the ec2 api
+    time: base.stats.types.Number,
+    bid: base.stats.types.Number,
+  },
+});
+
+// Store the times when a spot request is closed but not fulfilled
+module.exports.spotRequestsDied = new base.stats.Series({
+  name: 'AwsProvisioner.SpotRequestsDied',
+  columns: {
+    provisionerId: base.stats.types.String,
+    region: base.stats.types.String,
+    az: base.stats.types.String,
+    instanceType: base.stats.types.String,
+    workerType: base.stats.types.String,
+    id: base.stats.types.String,
+    // This is the true time per the ec2 api
+    time: base.stats.types.Number,
+    bid: base.stats.types.Number,
+    state: base.stats.types.String,
+    // These are the status fields
+    statusCode: base.stats.types.String,
+    statusMsg: base.stats.types.String,
```

Comment:
Same argument with `id` as above... I think `statusMsg` might be something we want to store. Or maybe we would just rather print it to the log... I'm we'll never really aggregate it...

Anyways, same argument as with `id`, do we need it and do we have use cases for this.

Note, as we probably don't have so many `spotRequestsDied` entries whatever might be fine :)